lua/treesitter-context.lua

```diff
@@ -92,16 +164,40 @@ function M.get_context(opts)
   return matches
 end
 
+function M.get_parent_matches()
+  if not parsers.has_parser() then return nil end
+
+  local contains = vim.tbl_contains
+
+  local matches = ts_query.get_capture_matches(0, '@scope.node', 'locals')
+
+  local curr_node = ts_utils.get_node_at_cursor()
+  if not curr_node then return end
```

Comment:
This can go at the start of the function no? It avoids calling `ts_query.get_capture_matches(0, '@scope.node', 'locals')`

Also remove newline between adjacent declarations.

Reply:
Oh also `current_node` instead of `curr_node`